superduperdb/base/config.py

```diff
@@ -161,7 +161,7 @@ class BytesEncoding(str, Enum):
 
 @dc.dataclass
 class Downloads(BaseConfig):
-    folder: t.Optional[str] = None
+    folder: t.Optional[str] = '/tmp/' #None
```

Comment:
If it has to be somewhere, it should be on the home directory of the user. like `~/.cache/superduperdb/downloads`